### DISCUSSION TITLE
KERI Implementer's Call 06/13/2024

### DISCUSSION THREAD

**Discussion by pfeairheller** [WebOfTrust/keri#88] [Meetings]
Reports
KERIpy - @m00sey (main), @pfeairheller (back up)
SignifyPy - @pfeairheller , @m00sey (back up)
KERIA - @rodolfomiranda , @2byrds (backup)
Signify-TS - @kentbull , @daniellenskjö (backup)
Cesride - @kentbull , @daidoji (backup)
Kerific - @kordwarshuis
WOT-Terms - @henkvancann
signify-browser-extension - @2byrds , @rodolfomiranda (backup)
polaris-web - @2byrds , @rodolfomiranda (backup)

Please add comments ahead of tomorrow's meeting for any additional topics

**Comment by rodolfomiranda** [comment]
## KERIA
No new issues
### PRs
- https://github.com/WebOfTrust/keria/pull/250
- https://github.com/WebOfTrust/keria/pull/253

**Reply by 2byrds** [reply]
KERIA dev tag needs a KERIpy dev tag. Currently the dependency is:
https://github.com/WebOfTrust/keripy/pull/804

**Comment by m00sey** [comment]
keripy

- https://github.com/WebOfTrust/keripy/pull/804
- https://github.com/WebOfTrust/keripy/pull/798
- https://github.com/WebOfTrust/keripy/pull/803
- https://github.com/WebOfTrust/keripy/pull/802
- https://github.com/WebOfTrust/keripy/pull/801
- https://github.com/WebOfTrust/keripy/pull/800
- https://github.com/WebOfTrust/keripy/pull/797

**Comment by edeykholt** [comment]
I could demo progress on the KERI Auth browser extension.  
https://github.com/KERIAuth/keriauth-blazor-wasm

**Comment by pfeairheller** [comment]
### SigPy

* No new PRs or Issues

**Comment by 2byrds** [comment]
polaris-web:
Would like to see https://github.com/WebOfTrust/polaris-web/pull/11 transition from draft to pull request.

signify-browser-extension:
Has been demonstrated with the above draft PR
